File: divisoresCadena.py
# https://omegaup.com/arena/problem/Divisores-de-una-cadena
import math;
cadena = input()
valor = 0
for el in cadena:
   valor += ord(el.upper())
divisores = 2;
for i in range(2,valor,1):
   if (valor % i == 0):
      divisores += 1
print(divisores);

File: Strings/divisoresCadena.cpp
//https://omegaup.com/arena/problem/Divisores-de-una-cadena
#include <bits/stdc++.h>

using namespace std;

int main(){
    string cadena;
    cin >> cadena;
    int total = 0;
    for(int i = 0; i < cadena.size(); i++){
        cadena[i] =toupper(cadena[i]);
        int letra = (int) cadena[i];
        total += letra;
    }
    int contador = 2;
    for(int i = 2; i <= (total / 2); i++){
        if(total % i == 0){
            contador++;
        }
    }
    cout << contador;
    return 0;
}